io.c

```diff
@@ -7374,6 +7374,9 @@ static VALUE popen_finish(VALUE port, VALUE klass);
  *  Executes the given command +cmd+ as a subprocess
  *  whose $stdin and $stdout are connected to a new stream +io+.
  *
+ *  This method has potential security vulnerabilities;
```

Comment:
I would change `potential security vulnerabilities` to `potential security vulnerabilities if called with untrusted input` in all cases.

Reply:
Done.